Dockerfile

```diff
@@ -15,28 +15,67 @@ RUN pip install "poetry==$POETRY_VERSION"
 FROM base as production
 # Copy only requirements to cache them in docker layer
 WORKDIR /todo_app
-COPY poetry.lock pyproject.toml .
+COPY poetry.lock pyproject.toml ./
 RUN poetry config virtualenvs.create false --local && poetry install --no-dev --no-root
 RUN poetry add gunicorn
 
 
 # Creating folders, and files for a project:
-COPY . /todo_app
+COPY . /todo_app 
 
 EXPOSE 5001
 ENTRYPOINT ["./gunicorn.sh"]
 
 #development
 FROM base as development
 WORKDIR /todo_app
-COPY poetry.lock pyproject.toml .
+COPY poetry.lock pyproject.toml ./
 RUN poetry config virtualenvs.create false --local && poetry install --no-root
 
 
 # Creating folders, and files for a project:
-COPY . /todo_app
+COPY . /todo_app ./
 
 
 EXPOSE 5001
 
 ENTRYPOINT ["poetry", "run","flask", "run", "--host", "0.0.0.0"]
+
+# testing stage
+FROM base as test
+COPY poetry.lock pyproject.toml ./
+
+RUN poetry config virtualenvs.create false --local && poetry install --no-dev --no-root
+COPY . /todo_app 
+COPY  . /.env.test
```

Comment:
This command doesn't look right - you're copying over everything into a folder named `.env.test` - is this meant to be `COPY .env.test /.env.test`? The .env.test file is actually already copied over as part of the above line, so we can get rid of this one

Reply:
Updated,thank you